get-help/index.html

```diff
@@ -7,21 +7,18 @@
 
 <div class="cb teaser">
     <article>
-        <h1>Get help</h1>
+       <h1>Get help</h1>
         <p>
-            Did you stumble upon a bug or run into problems while using Hoodie? Do you need support for trying it out? We’re here to help you.
+            Are you blocked with a bug you’re trying to fix, or a new feature you’re working on? Do you want to report harassment or a code of conduct violation? You’ve come to the right place!
         </p>
     </article>
 </div>
 <div class="cb">
     <article>
-        <h2>For any question you may have</h2>
-        <ul>
-            <li>Check our <a href="http://faq.hood.ie" target="_blank">FAQ</a> or</li>
-            <li><a href="https://github.com/hoodiehq/faq/issues/new" target="_blank">Send us a new question</a>. Having those questions in one place helps us make this content accessible for everyone in the community. If you don’t have a GitHub account, please <a href="/contact">contact us</a>.</li>
-            <li>Report a bug <a href="/bug">here</a>.</li>
-            <li>Maybe a <strong>Cheat Sheet</strong> can help you while you're getting familiar with Hoodie? You can get it here: <a href="/dist1/presentations/hoodie-cheat-sheet.pdf">Web Version</a> | <a href="/dist1/presentations/hoodie-cheat-sheet-print.pdf">Print Version</a>.
-        </ul>
+        <h2>Have questions about using Hoodie?</h2>
+        <p>
+        	We’re in the process of making our documentation even better, so if you have a question related to using Hoodie, join our public <a href="/chat">Slack</a> community and ask for help in the <strong>#help<strong> channel!
```

Comment:
the `<strong>` tag doesn’t close correctly. `<strong>#help<strong>` should be `<strong>#help</strong>`